roq-management/src/test/java/org/roqmessaging/management/zookeeper/RoQZooKeeperClientTest.java

```diff
@@ -281,4 +287,45 @@ public void testScalingConfig() {
 		
 		assertTrue(sc2.equals(sc1));
 	}
+	
+	/**
+	 * Actual tests
+	 */
```

Comment:
You can remove this comment. It was meant to separate the setup/teardown from the actual test code.